launch/camerav2_1280x960.launch

```diff
@@ -4,10 +4,14 @@
   <arg name="camera_id" default="0"/>
   <arg name="camera_frame_id" default="raspicam"/>
   <arg name="camera_name" default="camerav2_1280x960"/>
+  <arg name="vFlip" default="false"/>
+  <arg name="hFlip" default="false"/>
```

Comment:
Maybe just to make it more consistent with other params you can rename it from `vFlip` to eg. `vertical_flip` in launch files.